### DISCUSSION TITLE
New ideas

### DISCUSSION THREAD

**Discussion by cuttyflam04** [ThePandaOliver/Pandas-Extra-Details#5] [Ideas]
Panda there is a mod called fancy block particle which is discontinued in 1.12.2 the features are 3D block particles and a cool effect when placing a block
https://legacy.curseforge.com/minecraft/mc-mods/fancy-block-particles

**Comment by ThePandaOliver** [comment]
I think this goes too far from vanilla to fit in my mod, but if you think its something people would want, then i might code a new mod in the future.
I actually also like this mod.
